src/main/java/de/tum/in/www1/artemis/repository/LearningGoalRepository.java

```diff
@@ -36,6 +36,15 @@ public interface LearningGoalRepository extends JpaRepository<LearningGoal, Long
             """)
     Optional<LearningGoal> findByIdWithLectureUnitsBidirectional(@Param("learningGoalId") long learningGoalId);
 
+    @Query("""
+            SELECT learningGoal
+            FROM LearningGoal learningGoal
+            LEFT JOIN FETCH learningGoal.lectureUnits lu
+            LEFT JOIN FETCH lu.completedUsers
+            WHERE learningGoal.id = :#{#learningGoalId}
```

Comment:
```suggestion
            WHERE learningGoal.id = :learningGoalId
```
Does this not work?

Reply:
Yes, this should work. However, we use this Syntax throughout Artemis. If we change it, then for all occurrences and in a separate PR. :)